project3/styles.css

```diff
@@ -0,0 +1,218 @@
+:root{
+  --color-red:
+}
+*, 
+*::before,
+*::after{
+  box-sizing: border-box;
+}
+
+body{
+  background-color: #163346;
+}
+
+#Shield {
+    height: 239px;
+    width: 240px;
+    color: #000000;
```

Comment:
```suggestion
```
The `color` property only applies to text and, since this is an SVG, it's not meaningful here.